scripts/alert_emails/readme.md

```diff
@@ -10,15 +10,20 @@ We use handlebars to generate the email and send it via CreateSend.
 
 ## Sending the Alerts via Github Actions
 
-A github action will run all of these steps on github for you. Run that via
+A github action will run all of these steps on github for you.
 
-To create a draft of the emails that will be sent run:
+You must first get the current snapshot to use (from the modeling team). Confirm the lastSnapshot field is
```

Comment:
Can you change this to something like:

> The script should be run after a new snapshot has been pushed to the prod website, in order to notify users of changes.  You can get the currently live snapshot from https://github.com/covid-projections/covid-projections/blob/master/src/assets/data/data_url.json

In retrospect, we could make the script do this automatically...  but low-pri.  Added to the todo doc.